libs/libappindicator/libappindicator-gtk3-introspection-12.10.json

```diff
@@ -0,0 +1,129 @@
+{
```

Comment:
This one neither as the app seems to be using gtk2.

Reply:
Done